libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.scss

```diff
@@ -11,34 +100,146 @@ sky-colorpicker.sky-margin-stacked-lg {
+    )
+  );
+  border: none;
+  border-radius: var(
+    --sky-override-colorpicker-button-border-radius,
+    var(--sky-border-radius-s)
+  );
+  box-shadow: var(
+    --sky-override-colorpicker-button-box-shadow,
+    inset 0 0 0 var(--sky-border-width-input-base)
+      var(
+        --sky-override-colorpicker-button-border-color,
+        var(--sky-color-border-input-base)
+      ),
+    inset 0 0 0 var(--sky-colorpicker-border-padding-size)
+      var(--sky-color-background-container-base),
+    inset 0 0 0 var(--sky-color-picker-inner-border-size)
+      var(
+        --sky-override-colorpicker-button-border-color,
+        var(--sky-color-border-input-base)
+      )
+  );
   cursor: pointer;
+  position: relative;
+
+  &:hover {
+    box-shadow: var(
+      --sky-override-colorpicker-button-box-shadow,
+      inset 0 0 0 var(--sky-border-width-input-hover)
+        var(--sky-color-border-input-hover),
+      inset 0 0 0 var(--sky-colorpicker-border-padding-size)
+        var(--sky-color-background-container-base),
+      inset 0 0 0 var(--sky-color-picker-inner-border-size)
+        var(
+          --sky-override-colorpicker-button-border-color,
+          var(--sky-color-border-input-base)
+        )
+    );
+  }
+
+  &:focus-visible:not(:active) {
+    box-shadow: var(
+      --sky-override-colorpicker-button-box-shadow,
+      inset 0 0 0 var(--sky-border-width-input-focus)
+        var(--sky-color-border-input-focus),
+      inset 0 0 0 var(--sky-colorpicker-border-padding-size)
+        var(--sky-color-background-container-base),
+      inset 0 0 0 var(--sky-color-picker-inner-border-size)
+        var(
+          --sky-override-colorpicker-button-border-color,
+          var(--sky-color-border-input-base)
+        ),
+      var(--sky-elevation-focus)
+    );
+    outline: var(--sky-override-colorpicker-button-focus-outline);
+  }
+
   &.sky-colorpicker-button-disabled {
     cursor: default;
+    background-color: var(
+      --sky-override-colorpicker-disabled-background,
+      var(--sky-color-background-input-disabled)
+    ) !important;
+    box-shadow: var(
+      --sky-override-colorpicker-button-box-shadow,
+      inset 0 0 0 var(--sky-border-width-input-disabled)
+        var(
+          --sky-override-colorpicker-button-border-color,
+          var(--sky-color-border-input-disabled)
+        ),
+      inset 0 0 0 var(--sky-colorpicker-border-padding-size)
+        var(--sky-color-background-input-disabled),
+      inset 0 0 0 var(--sky-color-picker-inner-border-size)
+        var(
+          --sky-override-colorpicker-button-border-color,
+          var(--sky-color-border-input-disabled)
+        )
+    );
+
+    .sky-colorpicker-button-modern-chevron {
+      background-color: var(
+        --sky-override-colorpicker-disabled-background,
+        var(--sky-color-background-input-disabled)
+      );
+    }
+
+    .sky-colorpicker-button-icon {
+      color: var(--sky-color-text-deemphasized) !important;
```

Comment:
Same as above ^ it is set inline to a contrasting color when a color is selected